f/fluent-bit/fluent-bit-v2.0.11-ubi8.6.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	: fluent-bit
+# Version	: v2.0.11
+# Source repo	: https://github.com/fluent/fluent-bit
+# Tested on	: UBI 8.6
+# Language      : C++
+# Travis-Check  : true
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Sumit Dubey <sumit.dubey2@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+trap cleanup ERR
+function cleanup {
+  echo "ERROR: Script failed"
+}
+
+FLUENTBIT_VERSION=${1:-v2.0.11}
+USE_CENTOS_REPOS=${2:-1}
```

Comment:
@sumitd2 why are we keeping it configurable? I understand that not everyone would want to enable centos repos, but then won't the build fail or are these packages available in RHEL/EPEL?

also looks like with these variable definitions, we are diverging a bit from our standard naming conventions of PACKAGE_NAME, PACKAGE_VERSION etc.

Reply:
I am sure there are RHEL repos which have the required packages, so let the user enable them from outside the script. Variables definitions are fixed.